misc/python/materialize/cloudtest/util/web_request.py

```diff
@@ -39,16 +39,18 @@ def __init__(
         base_url: str,
         client_cert: tuple[str, str] | None = None,
         additional_headers: dict[str, str] | None = None,
+        default_timeout_in_sec: int = 15,
```

Comment:
maybe we should allow this to be set to `None` too, since otherwise there is no way to configure a request to have no timeout (since `self._timeout_or_default`) will ignore a `None` passed to it and fallback to this

Reply:
This was previously also not possible. But if we think that this will be beneficial, I can definitively adjust it.

Reply:
Ah good point. I don't think it's necessary then; we can add it later if necessary